app/src/main/java/ru/vssemikoz/newsfeed/data/mappers/ApiResponseMapper.java

```diff
@@ -2,18 +2,26 @@
 
 import java.util.ArrayList;
 import java.util.List;
-import java.util.Objects;
+
+import javax.inject.Inject;
 
 import retrofit2.Response;
 import ru.vssemikoz.newsfeed.models.Category;
 import ru.vssemikoz.newsfeed.models.NewsApiResponse;
 import ru.vssemikoz.newsfeed.models.NewsApiResponseItem;
 import ru.vssemikoz.newsfeed.models.NewsItem;
+import ru.vssemikoz.newsfeed.models.Params;
+
+public class ApiResponseMapper implements NewsMapper<Params> {
```

Comment:
What's happened with the naming? What does `ApiResponseMapper` do? We should choose a better, more meaningful name like `NewsApiResponseMapper` or even `NewsApiMapper`.